tests/unit/handler.unitTest.ts

```diff
@@ -40,17 +40,18 @@ describe("The lambda function handler", () => {
         expect(TechRecordsService.prototype.getTechRecordsList).toHaveBeenCalled();
       });
 
-      it("should call the /vehicles/{vin}/download-file/{filename} function with correct event payload", async () => {
+      it("should call the /vehicles/{vin}/download-file?filename=someFilename.pdf function with correct event payload", async () => {
```

Comment:
"should call the downloadFile function with correct payload" - the path is determined here, not in the function itself.

Reply:
Also still wrong. "/vehicles/{vin}/download-file?filename=someFilename.pdf" is an endpoint, not a  function - the function that you're actually testing is correctly invoked is the "downloadFile" function

Reply:
done